packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/import/vsphere-user-import.html

```diff
@@ -1,8 +1,8 @@
 <div
     data-wizard
     data-wizard-title=":: 'dedicatedCloud_USER_import_title' | translate"
-    data-wizard-on-cancel="$ctrl.goBack.bind($ctrl)"
-    data-wizard-on-finish="$ctrl.close.bind($ctrl)"
+    data-wizard-on-cancel="$ctrl.onCancelClick.bind($ctrl)"
+    data-wizard-on-finish="$ctrl.onCloseClick.bind($ctrl)"
```

Comment:
Hmm I'm not sure about that
`manager/packages/components/ng-ovh-utils/src/wizard/wizard-directive.jsL156`
```javascript
      $scope.onCancel =
        $attr.wizardOnCancel &&
        angular.isFunction($scope.$eval($attr.wizardOnCancel))
          ? $scope.$eval($attr.wizardOnCancel)
          : angular.noop;
```
It seems the expression is evaluated at each digest cycle
I recommand to make the binding during the controller's initialisation using a variable
```suggestion
    data-wizard-on-cancel="$ctrl.boundOnCancelClick"
    data-wizard-on-finish="$ctrl.boundOnCloseClick"
```

Reply:
```javascript
  constructor($q, $translate, DedicatedCloud) {
    this.$q = $q;
    this.$translate = $translate;
    this.DedicatedCloud = DedicatedCloud;
    this.boundOnCancelClick = this.onCancelClick.bind(this)
    this.boundOnCloseClick = this.onCloseClick.bind(this)
  }
```

Reply:
It's applied.